codecparsers/h265Parser.cpp

```diff
@@ -262,29 +291,29 @@ bool Parser::profileTierLevel(ProfileTierLevel* ptl, NalReader& nr,
         || ptl->general_profile_compatibility_flag[4]
         || ptl->general_profile_compatibility_flag[5])
         //The number of bits in this syntax structure is not affected by this condition
-        ptl->general_inbld_flag = nr.read(1);
+        READ(nr, ptl->general_inbld_flag);
     else
         nr.skip(1); // general_reserved_zero_bit
-    ptl->general_level_idc = nr.read(8);
+    READ_BITS(nr, ptl->general_level_idc, 8);
```

Comment:
@xuguangxin  Sorry, i am confused, if that is the case, how can i know when to use READ_BITS and when to use READ, since all of them are originally br/nr.read(x), (x>1).